deploy/roles/r-shiny-config/templates/shiny-server.conf.j2

```diff
@@ -31,6 +31,9 @@ server {
     run_as :HOME_USER:;
     user_dirs;
 
+    # Restrict to only allow certain groups to host peronal applications.
+    members_of {{ r_shiny_groups }};
```

Comment:
@rioh Have you tested this blank?  I'm just worried it might prevent anyone from hosting an app if there is no value there.  It might make sense to add a conditional around this, saying `if r_shiny_groups != ""`?